submissions/Jemikalajah-Philip/javascript/homepage.js

```diff
@@ -0,0 +1,41 @@
+let imgDiv = document.getElementsByClassName("product--image");
```

Comment:
`product--image`  a single hyphens is enough to separate class names

Reply:
There is no `product--image` anywhere else, not sure what this selector targets

Reply:
This is style of naming  was suggested  to me by one our mentors, Kunle.